src/zjs_buffer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "module":"buffer",
+    "constructor":"Buffer",
+    "targets":["arduino_101", "frdm_k64f"],
```

Comment:
I'm not sure if you can solve this now, but this breaks scalability because we have to list every supported platform. Perhaps if there is no "targets" it could mean all platforms should work?